pkg/kubernetes/Makefile.am

```diff
@@ -37,6 +39,7 @@ kubernetes_BUNDLE = \
 	pkg/kubernetes/deploy.min.js \
 	pkg/kubernetes/d3.min.js \
 	pkg/kubernetes/graphs.min.js \
+        pkg/kubernetes/nulecule.min.js \
```

Comment:
Please use tabs in Makefile.am. It's a strange thing with makefiles and we try to be consistent about tabs there :s